src/init.cpp

```diff
@@ -879,7 +879,7 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
     int64_t nMempoolSizeLimit = GetArg("-maxmempool", DEFAULT_MAX_MEMPOOL_SIZE) * 1000000;
     int64_t nMempoolDescendantSizeLimit = GetArg("-limitdescendantsize", DEFAULT_DESCENDANT_SIZE_LIMIT) * 1000;
     if (nMempoolSizeLimit < 0 || nMempoolSizeLimit < nMempoolDescendantSizeLimit * 40)
-        return InitError(strprintf(_("-maxmempool must be at least %d MB"), GetArg("-limitdescendantsize", DEFAULT_DESCENDANT_SIZE_LIMIT) / 25));
+        return InitError(strprintf(_("-maxmempool must be at least %d MB"), std::ceil(GetArg("-limitdescendantsize", DEFAULT_DESCENDANT_SIZE_LIMIT) / 25.0)));
```

Comment:
I don't like repeating the computation here in a different form. This is bound to give inconsistencies when it is changed later.
Can we do
```
int64_t mempoolSizeLimitMin = nMempoolDescendantSizeLimit * 40;
if (nMempoolSizeLimit < nMempoolSizeLimitMin)
    return InitError(strprintf(_("-maxmempool must be at least %d MB"), std::ceil(mempoolSizeLimitMin / 1000.0)));
```